CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Features:
 - [#1340](https://github.com/rails-api/active_model_serializers/pull/1340) Add support for resource-level meta. (@beauby)
 
 Fixes:
+- [#1622](https://github.com/rails-api/active_model_serializers/pull/1622) Fix fragment caching inherited serializers to use distinct per-serializer caches. (@lserman)
```

Comment:
Good point.  So, `Records are now fragment cached by serializer name,  rather that model name, allowing multiple fragment caches for a model.`